features/support/application_cuke_helpers.rb

```diff
@@ -32,6 +32,9 @@ def check_fields_validation_error(field_list)
     end
   end
 
+  def mobile_version?
```

Comment:
I was searching for something better, but I don't think that mode -- mobile or desktop is stored somewhere in that context. It is stored in the user session data, but it's accessible from controllers, but not from the Capybara context. At the same time we have the `@mobile` tag in cukes, but as it seemed to me that it's rather switches the browser emulation mode and that is why mobile version is picked automatically. So I don't think that there is anything more reliable than just getting it from the metadata in the page content, if we need to do that at all.